### DISCUSSION TITLE
Linking GSF Meetups with existing meetups

### DISCUSSION THREAD

**Discussion by danielvaughan** [Green-Software-Foundation/community_wg#47] [General]
Hi, following on from yesterday's discussion about GSF Meetups could we also consider how to tap into existing meetups to assess interest for dedicated groups.

For example I am a co-organizer for a Google Developers Group and there is a great network of GDG and GDSC (student club) organizers, 

Could we perhaps develop a talk on Green Software/GSF for a Google Developer/Google Cloud audience and use that to get people interested in becoming involved further. If we had one well developed talk we could half that on the shelf for engaging with those communities.

Perhaps there are other common communities we could produce tailored talks for e.g. AWS, Java, Python etc.

**Comment by jawache** [comment]
I do like the idea of a crafting a talk deck which we can use for multiple things - I get asked on this often, I have one based on the principles content but it's a year old already. 

I like the idea of a catalog of talk decks, even if all we had was a folder full of random "green software talk decks" donated to us that's already a better starting point for someone than starting a talk with a blank deck.

@tadanna / @aecurrie / @tmcclell - can we add this to the Community WG agenda?

**Comment by tmcclell** [comment]
@NAMRATA-WOKE pointed out that we already have these items and maybe feel they can't use them. @adamj89 really encourage and capture the impact. If we're talking to groups about green software it helps us to organically grow GSF. Find a way of measuring our reach when we talk to groups. @adamj89 - has a sample form and would like to discuss at next group meeting. @aecurrie - proxy for capturing through podcast.

**Comment by adamj89** [comment]
Work on this is ongoing but will be ad-hoc. Closing the discussion for now.
